app/cdap/components/Connections/Browser/GenericBrowser/index.tsx

```diff
@@ -141,15 +145,23 @@ export function GenericBrowser({ selectedConnection }) {
           loading={loading}
         />
       </If>
-      <If condition={isEmpty && !error}>
-        <MuiAlert severity="info" className={classes.alert}>
-          No entities available
-        </MuiAlert>
+      <If condition={isEmpty && !loading}>
+        <EmptyMessageContainer title="No entities available">
+          <ul>
+            <li>
+              <span className="link-text" onClick={clearSearchString}>
+                {T.translate(`features.EmptyMessageContainer.clearLabel`)}
```

Comment:
What makes the difference between this string, which gets translated, and the strings on lines 155 and 158, which currently do not? Is this intentional?

Reply:
This should all be internationalized. Good catch.

Reply:
Updated with full I18n for this portion.